lib/adapters/amqp/index.ts

```diff
@@ -10,7 +10,7 @@ import {AMQPPublisherAdapter} from "./publisher";
 import {AMQPSubscriberAdapter} from "./subscriber";
 
 const AMQP = require("amqp-coffee");
-const _ = require("lodash");
+const clone = require("lodash.clone");
```

Comment:
Config can be something different from a plain object (could contain nested objects and/or arrays). 
I can't ensure that `Object.assign` could be used here. If anyone can ensure that, I'll change that